pkg/process/exec_conf.go

```diff
@@ -195,3 +197,40 @@ func cleanup(cmd *cobra.Command) {
 		return err
 	}
 }
+
+// ApplicationDir returns best base directory for specific OS
+func ApplicationDir(subdir ...string) string {
```

Comment:
Let's put this code into `fpath` rather than `process`. `fpath` seems more concerned with filepath handling than `process`.

Reply:
done